editor/scene_tree_dock.cpp

```diff
@@ -84,10 +84,13 @@ void SceneTreeDock::_inspect_hovered_node() {
 	TreeItem *item = tree->get_item_with_metadata(node_hovered_now->get_path());
 	if (item) {
 		if (tree_item_inspected) {
+			tree_item_inspected->remove_meta(SNAME("custom_color"));
 			tree_item_inspected->clear_custom_color(0);
 		}
 		tree_item_inspected = item;
-		tree_item_inspected->set_custom_color(0, get_theme_color(SNAME("accent_color"), EditorStringName(Editor)));
+		Color accent_color = get_theme_color(SNAME("accent_color"), EditorStringName(Editor));
+		tree_item_inspected->set_custom_color(0, accent_color);
+		tree_item_inspected->set_meta(SNAME("custom_color"), accent_color);
```

Comment:
This duplicates `_set_item_custom_color()` in SceneTreeEditor.
Maybe it could be made public. Or maybe for one usage it does not matter.

Reply:
I'd have to add a remove_custom_color as well then. I can do that if you want!